### DISCUSSION TITLE
Filter start when press a specific button, not automatically

### DISCUSSION THREAD

**Discussion by R9X3R** [laravel/nova-issues#3450] [Q&A]
I am working on a project in Laravel Nova and I would like that when I press on the options of a Filter, the filtering does not happen at the click/tap (because some queries take too long time) but only at a later time, at the press of a specific button "apply filters". How can I do this?

**Comment by crynobone** [comment]
This is not possible at the moment by default. Not sure if there any 3rd party package offering it.
